### PR TITLE
Created truncated game and game_teams files

### PR DESCRIPTION
**Pull request by whitneykidd**
Made a folder within the test for the truncated data.

### CODE REVIEW

File: test/fixtures/game_teams_truncated.csv [hunk 1]
```diff
@@ -0,0 +1,300 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```
Comment (tylertomlinson) [inline test/fixtures/game_teams_truncated.csv:1]: Nice! Going to make testing much easier.

Comment (Maxwell-Baird) [inline test/fixtures/game_teams_truncated.csv:1]: I thought we need all three data files

Comment (alex-latham) [reply test/fixtures/game_teams_truncated.csv:1]: The teams file is pretty short already (33 lines)

Comment (alex-latham) [reply test/fixtures/game_teams_truncated.csv:1]: Could we make these shorter? So it's easier to manually calculate things like percentage_home_wins for our testing.

File: test/fixtures/game_teams_truncated.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+2012030221,6,home,WIN,OT,Claude Julien,3,12,51,6,4,1,55.2,4,5
+2012030222,3,away,LOSS,REG,John Tortorella,2,9,33,11,5,0,51.7,1,4
+2012030222,6,home,WIN,REG,Claude Julien,3,8,36,19,1,0,48.3,16,6
+2012030223,6,away,WIN,REG,Claude Julien,2,8,28,6,0,0,61.8,10,7
+2012030223,3,home,LOSS,REG,John Tortorella,1,6,37,2,2,0,38.2,7,9
+2012030224,6,away,WIN,OT,Claude Julien,3,10,24,8,4,2,53.7,8,6
+2012030224,3,home,LOSS,OT,John Tortorella,2,8,40,8,4,1,46.3,9,7
+2012030225,3,away,LOSS,REG,John Tortorella,1,7,25,13,2,1,50.9,5,3
+2012030225,6,home,WIN,REG,Claude Julien,3,8,35,11,3,1,49.1,12,9
+2012030311,6,away,WIN,REG,Claude Julien,3,7,19,17,4,0,66.7,1,5
```
Comment (Maxwell-Baird) [inline test/fixtures/game_teams_truncated.csv:12]: We are missing a team here for game_id 2012030311

**Pull request closed without merging**